### PR TITLE
project completed.

### PR DESCRIPTION
**Pull request by DilsherB**
Awesome books developed using ES6 approach.

### CODE REVIEW

**Comment by TedLivist** [comment]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @DilsherB,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly fix your ESLint error.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by DilsherB** [comment]
> ## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑
> 
> Hi @DilsherB,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> ## Invalid Code Review Request
> ### Kindly fix your ESLint error.
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

Thank you for your kind reply. I need your help in this regard. Actually these errors are for 'luxon.js' how to quiet them for a specific file like 'luxon.js'

**Comment by DilsherB** [comment]
@TedLivist 
Thank you for your kind reply. I need your help in this regard. Actually these errors are for 'luxon.js' how to quiet them for a specific file like 'luxon.js'

**Comment by TedLivist** [comment]
Hi @DilsherB you are free to disable linters in the external libraries files, like your `modules/luxon.js` file

**Comment by DilsherB** [comment]
> Hi @DilsherB you are free to disable linters in the external libraries files, like your `modules/luxon.js` file

but how can i disable it for luxon.js only

**Comment by TedLivist** [comment]
Hi @DilsherB you can add this comment to the top of the file

```
/* eslint-disable */
```

It should disable eslint for that particular file

File: index.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+document.querySelector('.date').innerHTML = `${dt.day} ${dt.month}, ${dt.year} ${dt.hour}:${dt.minute}:${dt.second}`;
+
+const menuItems = document.querySelectorAll('li');
+const sections = document.querySelectorAll('section');
+function reset() {
+  sections.forEach((section) => {
+    section.style.display = 'none';
+  });
+}
```
Comment (TedLivist) [inline index.js]: - [ ] In keeping with the ES6 syntax requirement, kindly refactor this function to an arrow function.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @DilsherB,

Good job so far!
✔️ Use of luxon package for date and time
✔️ Books can be added and removed
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (DilsherB) [reply index.js]: > * [ ]  In keeping with the ES6 syntax requirement, kindly refactor this function to an arrow function.

@TedLivist Thank you for pointing out reset function which was still to convert in arrow function. I have done and submitted again.

**Comment by rloterh** [comment]
@DilsherB 
Kindly fix linter errors asap!

File: index.js (hunk 1 above)
Comment (rloterh) [reply index.js]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Dilsher,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
- Code follows ES6 syntax
- Add/remove feature working as intended
- Good ReadMe

## Optional suggestions 🧰 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

**You can also consider:**
- Kindly endeavor to fill the FAQ section with at least 2 Q&A(s), or optionally remove this section.
- It's recommended as good practice that you add an MIT license or any other similar license to your ReadMe for this project.

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**